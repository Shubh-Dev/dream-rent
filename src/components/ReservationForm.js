import React from 'react';

function ReservationForm() {
  return (
    <div>ReservationForm</div>
  );
}

export default ReservationForm;
