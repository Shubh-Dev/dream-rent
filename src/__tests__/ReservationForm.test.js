import React from 'react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer';
import ReservationForm from '../components/ReservationForm';
import store from '../redux/configureStore';
import { Provider } from 'react-redux';

describe('ReservationForm', () => {
    it('renders correctly', () => {
        const tree = TestRenderer.create(
        <Provider store={store}>
            <ReservationForm />
        </Provider>,
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('display RESERVATION FORM', () => {
        const testRenderer = TestRenderer.create(
        <Provider store={store}>
            <ReservationForm />
        </Provider>,
        );
        const testInstance = testRenderer.root;
        const h1 = testInstance.findByType('h1');
        });
    });