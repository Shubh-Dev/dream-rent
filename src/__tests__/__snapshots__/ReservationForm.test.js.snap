// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationForm renders correctly 1`] = `
<h1>
  Loading...
</h1>
`;
